bin/verilator

```diff
@@ -398,6 +398,7 @@ detailed descriptions of these arguments.
     --no-skip-identical         Disable skipping identical output
     --stats                     Create statistics file
     --stats-vars                Provide statistics on variables
+    --std                       Verilate SystemVerilog's std package
```

Comment:
As you also noted, I don't want users to have to specify this.

My thinking was the parser would note a mailbox/std::mailbox/semaphore/std::semaphore went by, hack up the symbol tree, then after all other sources are done read the std.sv.

Reply:
Did something like this.